test/util.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# This utility file contains functions that wrap commands to be tested and allow us to interface 
+# our command-line tests with JUnit on Jenkins. All wrapper functions run commands in a sub-shell 
+# and redirect all output. Tests in test-cmd *must* use these functions for testing.
+
+# HIGH-LEVEL WRAPPER FUNCTIONS: ASSUMING EXIT CODES
+
+# expect_success runs the cmd and expects an exit code of 0
+function os::test::expect_success() {
```

Comment:
We have a whole set of scripts for building our releases, which live under `os::build`. We have a whole set of scripts which are utility for everyone (like OSX-compatible `sed`) that live under `os::util`.